File: WebApp/app/model/OrderElement.js
Ext.define('WebApp.model.OrderElement', {
    extend: 'Ext.data.Model',
    fields: [
        {name : 'OrderElementId'},
		{name : 'OrderId'}, 
		{name : 'ItemId'}, 
		{name : 'ItemsCount'}, 
        {name : 'ItemPrice'},
        {name : 'Item', mapping: 'Item'},
        {name : 'Item.Code', mapping: 'Item.Code'},
        {name : 'Item.Name', mapping: 'Item.Name'}
    ],
	hasOne : {
        model : 'Item',
        associationKey : 'item'
    },
	hasOne : {
        model : 'Order',
        associationKey : 'order'
    }
});

File: WebApp/Models/OrderElement.cs
namespace WebApp.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("OrderElement")]
    public partial class OrderElement
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Key]
        public Guid OrderElementId { get; set; }

        public Guid OrderId { get; set; }

        public Guid ItemId { get; set; }

        public int ItemsCount { get; set; }

        public int ItemPrice { get; set; }

        public virtual Item Item { get; set; }

        public virtual Order Order { get; set; }
    }
}
